backend/daemon/daemon_e2e_test.go

```diff
@@ -842,17 +842,354 @@ func TestSubscriptions(t *testing.T) {
 	require.NoError(t, err)
 	time.Sleep(time.Millisecond * 100)
 
-	docGotten, err := bob.RPC.DocumentsV3.GetDocument(ctx, &documents.GetDocumentRequest{
+	require.Eventually(t, func() bool {
+		docGotten, err := bob.RPC.DocumentsV3.GetDocument(ctx, &documents.GetDocumentRequest{
+			Account: aliceHome.Account,
+			Path:    aliceHome.Path,
+		})
+		require.NoError(t, err)
+		return len(aliceHome.Content) == len(docGotten.Content)
+	}, time.Second*2, time.Millisecond*100, "We should have three comments, including carol's")
```

Comment:
@juligasa is this error message accurate? We query bob here for Alice's document, then we check that we receive the same version, while the error message is talking about three *comments* and says something about Carol. Is this correct?